nns/nns_contract.go

```diff
@@ -113,6 +139,39 @@ func _deploy(data interface{}, isUpdate bool) {
 	storage.Put(ctx, []byte{prefixRegisterPrice}, defaultRegisterPrice)
 }
 
+// remove after v0.13.1 upgrade
+func shBEFromLEHex(s string) interop.Hash160 {
+	res := make([]byte, interop.Hash160Len)
+
+	ln := len(s) / 2
+	for i := 0; i < ln; i++ {
+		a := hexToNum(s[i*2])
```

Comment:
```
// don't do ln := len(s) / 2
for i := 0; i < ln; i += 2 {
  a := hexToNum(s[i])
  b := hexToNum(s[i+1])
  ...
}
```
is cheaper isn't it?

Reply:
I use `i` to address the byte in the resulting slice

```go
res[ln-1-i] = n.([]byte)[0]
```

In your example I will need to divide on each iteration to specify correct index of `res`.